source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,53 @@ title: "Onboarding Home Assistant"
 description: "Instructions to get Home Assistant configured."
 ---
 
-Alright, you made it here. The tough part is done.
+After your Home Assistant server is up and running, there are 5 simple steps to log in and see your devices.
 
-- As the last step of the [installation procedure](/installation/) specific to your device, you entered the Home Assistant device's address in your browser's address bar.
-    - Typically, this is `http://homeassistant.local:8123/`.
-    - If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at <a href="http://homeassistant:8123" target="_blank">homeassistant:8123</a> or `http://X.X.X.X:8123` (replace X.X.X.X with your device's IP address).
+**Step 1**: type  <http://homeassistant.local:8123/> into your browser.
+![Home Assistant preparation](/images/getting-started/onboarding_preparing_01_.png)
 
-    - **Result**: You now see the welcome screen. Depending on your hardware, preparation may take a while.
+{% details If that didn’t work %}
+If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at homeassistant:8123 or http://X.X.X.X:8123 (replace X.X.X.X with your device’s IP address).
```

Comment:
- `homeassistant:8123` looks odd here
- Same for the X.X.X.X URL that lost formatting and is now auto linked to something non-existing.

Reply:
Both of those were in previous version. I didn't touch it. Can anyone who check in the previous version comment here?

Reply:
The formatting got lost, so it is not the same as the previous version.

Reply:
Ah, ok, I'll check that out, thanks.